packages/angular/src/schematics/library/schema.json

```diff
@@ -80,12 +76,19 @@
     "routing": {
       "type": "boolean",
       "default": false,
-      "description": "Add router configuration. See lazy for more information."
+      "description": "Add router configuration. See lazy for more information.",
+      "x-prompt": "Would you like to configure routing for this library?"
```

Comment:
I think it makes a lot of sense to ask for routing when creating applications: most angular applications use the router. 

But I'm not sure if same is true for libraries. At least half of libraries (utils, ui, data access etc) won't have the router on. So asking the users to answer three extra questions doesn't seem ergonomic.

What do you think?

Reply:
That's fair, most libraries will not need it. I'm not sure as well. The way I see it, it's a trade off:

1. If it's not there and it's forgotten
  a. The changes have to be reverted
  b. The user has to go through the entire flow again and type in the options manually as well.
2. If it's there and it's not needed most of the time
  a. The user will have a minor nuisance and need to press enter 3 times.

> Better to have, and not need, than to need, and not have. - Franz Kafka 🤷‍♀

This could really help us: https://github.com/angular/angular-cli/issues/13383

Reply:
yeah, I'd remove xprompt for now. If it were a single field, we could consider having the xprompt in.  In general, I think doing less but without irritating people is better than doing more and being an annoyance two times out of three.

Reply:
Done